cmd/av/pr_queue.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/aviator-co/av/internal/actions"
+	"github.com/aviator-co/av/internal/avgql"
+	"github.com/shurcooL/graphql"
+	"github.com/spf13/cobra"
+)
+
+var prQueueFlags struct {
+	SkipLine 	bool
+	Targets		string
+}
+
+var prQueueCmd = &cobra.Command{
+	Use:          "queue",
+	Short:        "queue a pr for the current branch",
+	SilenceUsage: true,
+	Args:         cobra.NoArgs,
+	// error or reterr from emperror.dev/errors here?
+	RunE: func(cmd *cobra.Command, args []string) error {
+		variables, err := getMutationVariables()
+		if err != nil {
+			return err
+		}
+
+
+		// I have a feeling this would be better written inside of av/internals
+		client := avgql.NewClient()
+
+		var mutation struct {
+			QueuePullRequest struct {
+				QueuePullRequestPayload struct {
+					PullRequest	graphql.String
+				} `graphql:"... on QueuePullRequestPayload"`
+					
+			} `graphql:"queuePullRequest(input: {repoOwner: $repoOwner, repoName:$repoName, number:$prNumber})"`
+		}
+		err = client.Mutate(context.Background(), &mutation, variables)
+		if err != nil {
+			return fmt.Errorf("client.Mutate Error %#+v", err)
+		}
+		// We print to stderr everywhere, why?
+		// Also, should we surface anything else to the user? bar improving what we actually 
+		pr := mutation.QueuePullRequest.QueuePullRequestPayload.PullRequest
+
+		fmt.Fprintf(
+			os.Stdout, 
+			string(pr),
+			"\n",
+		)
+		return nil
+	},
+}
+
+func getMutationVariables() (map[string]interface{}, error) {
+	repo, err := getRepo()
+	if err != nil {
+		return nil, err
+	}
+
+	db, err := getDB(repo)
+	if err != nil {
+		return nil, err
+	}
+
+	tx := db.ReadTx()
+
+	currentBranchName, err := repo.CurrentBranchName()
+	if err != nil {
+		return nil, err
+	}
+
+	branch, _ := tx.Branch(currentBranchName)
+
+	if branch.PullRequest == nil {
+		return nil, errors.New(
+			"this branch has no associated pull request (run `av pr create` to create one)",
+		)
+	}
+
+	prNumber := branch.PullRequest.Number
+	repository, exists := tx.Repository()
+	if !exists {
+		return nil, actions.ErrRepoNotInitialized
+	}
+
+	var variables = map[string]interface{}{
+		"repoOwner": graphql.String(repository.Owner),
+		"repoName":  graphql.String(repository.Name),
+		"prNumber":  graphql.Int(prNumber),
+	}
+	return variables, nil
+}
+
+func init() {
+	prQueueCmd.Flags().BoolVar(
+		&prQueueFlags.SkipLine, "skip-line", false,
+		"skip in front of the existing PRs, merge this PR right now",
```

Comment:
nit: let's always use "pull request" rather than "PR" in user-facing interfaces

```suggestion
		"queue this pull request ahead of all others in the queue",
```

Reply:
Done